src/auro-toast.js

```diff
@@ -81,30 +103,79 @@ export class AuroToast extends LitElement {
     const toastContainer = this.shadowRoot.querySelector('#toastContainer');
     toastContainer.className = 'hidden';
 
-    // Wait until fade out completes to set visible to false
     setTimeout(() => {
-      this.visible = false;
+      this.closeToast();
     }, FADE_OUT_DURATION);
   }
 
-  updated() {
-    if (this.visible) {
+  /**
+   * @private
+   * @returns {void}
+   */
+  closeToast() {
+    clearTimeout(this.fadeOutTimer);
+    this.visible = false;
+    this.dispatchEvent(new CustomEvent('onToastClose', {
+      bubbles: true,
+      composed: true,
+      detail: this
+    }));
+  }
+
+  /**
+   * For mobile, set the onclick function so the toast can be dismissed if it is tapped on anywhere inside the toast.
+   * @private
+   * @returns {void}
+   */
+  setOnClick() {
+    const mobileBreakPoint = 767;
+    if (window.innerWidth < mobileBreakPoint) {
+      this.onclick = () => {
+        this.fadeOutToast();
+      };
+    }
+  }
+
+
+  updated(changedProperties) {
+    this.setOnClick();
```

Comment:
seOnClick will run every time the updated lifecycle runs- would connectedCallback be a more appropriate lifecycle to invoke this?

Reply:
i like that suggestion, i'll make sure to push some changes for this